ibm/service/schematics/resource_ibm_schematics_action.go

```diff
@@ -1072,14 +1071,6 @@ func resourceIBMSchematicsActionCreate(context context.Context, d *schema.Resour
 		}
 		createActionOptions.SetSettings(settings)
 	}
-	if _, ok := d.GetOk("state"); ok {
```

Comment:
Is it similar change like sys_lock? then do we also have to have schema changes for state?

Doc changes for these two attributes are missing.

Reply:
I see that state is already computed. So ignore. :) 
Make sure the docs are updated right.